src/lib/ui/thermostat/ThermostatCard.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box } from '@mui/material'
+import type { Meta, StoryObj } from '@storybook/react'
+import type { ThermostatDevice } from 'seamapi'
+
+import { useDevice } from '../../../hooks.js'
+import { ThermostatCard } from './ThermostatCard.js'
+
+const meta: Meta<typeof ThermostatCard> = {
+  title: 'Library/ThermostatCard',
+  tags: ['autodocs'],
+  component: ThermostatCard,
+}
+
+type Story = StoryObj<typeof ThermostatCard>
+
+export const Content: Story = {
+  render: () => {
+    const { device } = useDevice({
+      device_id: 'device5',
+    })
+
+    return (
+      <Box display='grid' gap={4} gridTemplateColumns='repeat(4, min-content)'>
+        <ThermostatCard device={device as ThermostatDevice} />
+      </Box>
+    )
+  },
```

Comment:
```suggestion
  render: () => {
    const { device } = useDevice({
      device_id: 'device5',
    })
    
    if (!isThermostatDevice(device)) return null

    return (
      <Box display='grid' gap={4} gridTemplateColumns='repeat(4, min-content)'>
        <ThermostatCard device={device} />
      </Box>
    )
  },
```

Reply:
`isThermostatDevice` won't accept `null` or `undefined`, so I have to add some additional conditions here. (Should we update these functions to accept empty vals though?)

Reply:
Ah I see. No let's not change them to allow null. In this case, we can just update the story with the missing loading / error states. Really for the story you could put

```
if (device == null) return null
```

If you want to get fancy you could add a MUI loading spinner since the stories are allowed to use MUI.